testkit/src/main/scala/org/bitcoins/testkit/wallet/BaseWalletTest.scala

```diff
@@ -25,8 +25,12 @@ trait BaseWalletTest extends EmbeddedPg { _: Suite with BitcoinSAkkaAsyncTest =>
   }
 
   /** Wallet config with data directory set to user temp directory */
-  protected def getFreshConfig: BitcoinSAppConfig =
-    BaseWalletTest.getFreshConfig(pgUrl, Vector.empty)
+  protected def getFreshConfig: BitcoinSAppConfig = {
+    val bipPasswordOpt = KeyManagerTestUtil.bip39PasswordOpt
```

Comment:
there is where it get set now rather than being passed as a parameter everywhere